README.md

```
@@ -7,44 +7,211 @@
+svelteuml ./my-sveltekit-app
+
+# Specify output path
+svelteuml ./my-sveltekit-app -o architecture.puml
+
+# Generate SVG (requires Java + PlantUML)
+svelteuml ./my-sveltekit-app -f svg -o diagram.svg
 ```
 
 ## Usage
 
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)